modules/sc-mesh-secure-deployment/src/2_0/features/PHY/RSS_auth/F_RSS_Auth.py

```diff
@@ -0,0 +1,182 @@
+import csv
+import threading
+import subprocess
+import time
+from datetime import datetime
+from tabulate import tabulate
+import os
+
+class RSS_Auth:
+    def __init__(self, start_script, mac_address_file, rssi_file):
+        self.start_script = start_script
+        self.mac_address_file = mac_address_file
+        self.rssi_file = rssi_file
+        self.default_macs = []
+        self.current_macs = []
+        self.rssi_avg = []
+        self.updated_rssi_avg = []
+        self.threshold = 4
+        self.process = None
+        self.results = {'Pass': [], 'Fail': []}
+
+    def start(self):
+        """Starts the RSSI capturing process in a separate thread."""
+        # Starts the RSSI capturing script as a subprocess
+        self.process = subprocess.Popen(["python3", self.start_script], stdout=subprocess.PIPE)
```

Comment:
Why not importing and executing a function?

Reply:
Yes, you are right. if you are okay, I can do this modification for the next advance release of this solution. Otherwise, I will do the changes. Please advise